### ISSUE TITLE
[Bug]: Authentication Fail loop - iPad Pro 2

### ISSUE THREAD

**Issue by GtrNut** [denolehov/obsidian-git#460]
### Describe the bug

Following explicit guideline in README and this previous post - https://github.com/denolehov/obsidian-git/issues/305#issuecomment-1242979455

There seems to be an issue with authenticating via the command pallete/plugin settings. No matter what, creds are bad for the clone. In my troubleshooting I also created a brand new repo instead of using one that I have setup for my desktop sync, similar to the stresstest repo mentioned in README, but WAY smaller in size. 

### Steps to reproduce

Obsidian Git Version: 2.17.4 
Obsidian for ipad OS Version: 1.4.3 (95)

Follow setup guides linked above and readme file. Start the clone command from command pallete, and progress through popups. 1. URL
2. root area
3. does not contain .obsidian

Auth failure pops up after initializing, and then new pop ups arrive asking for your creds. Enter them in, still failure. Stuck in this loop until you close the initialization and "aborted" pops up.  

### Operating system

iOS

### Installation Method

None

### Plugin version

2.17.4

**Comment by Vinzent03** [comment]
Did you create a personal access token with the correct rights and used that as a password?

**Comment by quickcharge3** [comment]
hey @Vinzent03,

same exact problem here unfortunately.

I have generated a personal access token with the necessary rights and used it as a password. I have tried both with a fine grained token as well as a classic token.

I have tried with both existing vaults and fresh vaults multiple times.

If there's any more info I can provide please let me know.

Thank you

Obsidian Version
1.4.6 (105)

Plugin version
2.20.3

Device
iPhone Xs

Operating system
iOS 16.5

**Comment by quickcharge3** [comment]
I figured it out... I was cloning into the wrong folder... I was using vault root, making a new folder and cloning into that did work fine. Not sure why an auth error comes up in that case but anyway, it's working 😋

**Comment by jalberto** [comment]
Got same issue, it only worked with classic token for me

**Comment by Vinzent03** [comment]
@jalberto Please ensure you are setting it up correctly, because I honestly doubt there is a issue specifically to fine grained access token, because the handling on the client should be the same as with classic tokens. https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token

**Comment by jalberto** [comment]
I do not understand what's the issue either. I created 3 different fine-grained tokens from minimal requirements to all access, and none worked for me, then the classic one with the minimum permissions worked at first try.

The only difference is the expiration day, as in fine-grained is set to a specific date.

There is a small contradiction in the instructions compared to the issue where it was implemented, related to the username, as in the issue says to use `__token__` as username, I tried both anyway, same result.

Thanks for your plugin, it is great
